pkg/constants/constants.go

```diff
@@ -85,4 +85,7 @@ const (
 
 	// OktetoGitCommitEnvVar is the SHA1 hash of the last commit of the branch.
 	OktetoGitCommitEnvVar = "OKTETO_GIT_COMMIT"
+
+	// OktetoNamespaceLabel is the label used to identify the namespace where the resource lives
+	OktetoNamespaceLabel = "dev.okteto.com/namespace"
```

Comment:
What's the idea with this new constant? Should we include it in every resource? Or just on CRDs?

Reply:
Just for the external resources or any object in which we want to do queries by that. In this case, it is to optimize how we retrieve the endpoints for external resources from a list of namespaces